selfdrive/car/body/interface.py

```diff
@@ -16,7 +16,8 @@ def get_params(candidate, fingerprint=None, car_fw=None, disable_radar=False):
     ret.carName = "body"
     ret.safetyConfigs = [get_safety_config(car.CarParams.SafetyModel.body)]
 
-    ret.minSteerSpeed = -math.inf
+    ret.minSteerEnableSpeed = -math.inf
+    ret.minSteerDisableSpeed = -math.inf
```

Comment:
We can't infer disable from enable (if car enables and disables at one speed) if we only want to set one, we couldn't differentiate between an asymmetric steer speed car and a symmetric one.